docs/Release_Notes.md

```diff
@@ -31,60 +37,151 @@ Lastly, link the `ecflow` scripts by executing:
 
 Version File Changes
 --------------------
+New files: build.ver and run.ver
 
 
 Sorc Changes
 ------------
-
+No change from GFSv16
 
 Job Changes
-------------
+-----------
+Filename changes according to EE2 standards:
+ |   wafs.v7                   |    GFSv16                     
+ |   ------------------------- |    ----------------------------------------------------|
+ |   JWAFS_GCIP                |<- JGFS_ATMOS_WAFS_GCIP                                 |
+ |   JWAFS_GRIB                |<- JGFS_ATMOS_WAFS                                      |
+ |   JWAFS_GRIB2_0P25          |<- JGFS_ATMOS_WAFS_GRIB2_0P25                           |
+ |   JWAFS_GRIB2_0P25_BLENDING |<- JGFS_ATMOS_WAFS_BLENDING_0P25                        |
+ |   JWAFS_GRIB2_1P25          |<- JGFS_ATMOS_WAFS_GRIB2                                |
+ |   JWAFS_GFS_MANAGER (new)   |   JGFS_ATMOS_WAFS_BLENDING (removed after cleaning up) |
+ |   JWAFS_UPP         (new)   |                                                        |
 
 
 Parm Changes
 ------------
-
+1. Remove parm/wafs/legend
+2. parm/upp is created after system building. Per AWC request, WAFS UPP control files add 4 low levels for icing and 1 upper lower for turbulence:
+   - postxconfig-NT-GFS-WAFS.txt
+   -  postcntrl_gfs_wafs.xml
 
 Script Changes
 --------------
+First of all, filename changes according to EE2 standards:
+ |   wafs.v7                       |    GFSv16
+ |   ----------------------------- |    ---------------------------------------------------------|
+ |   exwafs_gcip.sh                |<- exgfs_atmos_wafs_gcip.sh                                  |
+ |   exwafs_grib2_0p25_blending.sh |<- exgfs_atmos_wafs_blending_0p25.sh                         |
+ |   exwafs_grib2_0p25.sh          |<- exgfs_atmos_wafs_grib2_0p25.sh                            |
+ |   exwafs_grib2_1p25.sh          |<- exgfs_atmos_wafs_grib2.sh                                 |
+ |   exwafs_grib.sh                |<- exgfs_atmos_wafs_grib.sh                                  |
+ |   exwafs_gfs_manager.sh (new)   |   exgfs_atmos_wafs_blending.sh (removed after cleaning up)  |
+ |   exwafs_upp.sh (new)           |                                                             |
+
+Additionally there are other changes:
+1. In exwafs_grib2_0p25_blending.sh, extend the waiting time window for UK data from 15 to 25 minutes
+2. In exwafs_grib2_0p25.sh, only include fields at the extra levels when forecast hour is between 06 and 36 per AWC request
+3. In ush/mkwfsgbl.sh: change input dependency from GFS pgrb2.1p00 to master file.
+4. Remove files under ush/ folder: wafs_blending.sh wafs_grib2.regrid.sh wafs_intdsk.sh
 
 
 Fix Changes
 -----------
-
+1. Remove fix/wafs/legend folder
+2. Remove files under fix/wafs:
+   - grib2_blended_wafs_wifs_fFF.0p25
+   - grib2_gfs_wafs_wifs_fFF.0p25
+   - grib_wafsgfs_intdsk
+   - grib_wafsgfs_intdskf00   
+3. Under fix/wafs, filenames are changed according to EE2 standards.
```

Comment:
fix files do not have a EE2 naming convention

Reply:
I will remove the words of "EE2 standards", while keeping fix/wafs filename changes